src/app/components/LanguageSources.svelte

```diff
@@ -1,11 +1,13 @@
-<script lang="ts">
-  import AddSource from "./AddSource.svelte";
-  interface DictionarySource {
+<script lang="ts" context="module">
+  export interface DictionarySource {
```

Comment:
This may be out of scope for this PR, but I'd like to [keep TypeScript code in Svelte to a minimum](https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#svelte-keep-script-sections-as-small-as-possible).

So I suggest extracting this interface to a dedicated `.ts` file, something like

```typescript
// src/app/types.ts
export interface DictionarySource { /* ... */ }
```

And then use it in Svelte files as:

```svelte
<script lang="ts">
  import type { DictionarySource } from "../types.ts";
```

**The `import type` is important!** Otherwise, Svelte gives you a misleading error message, like "cannot import DictionarySource" or "types.ts does not export DictionarySource", something like that.

Reply:
I think I've seen a few examples like this around, I'll give that a try!